### DISCUSSION TITLE
Split cell

### DISCUSSION THREAD

**Discussion by alievk** [marimo-team/marimo#2857] [Ideas]
Splitting a cell at the cursor position is a very useful operation which helps to organize your notebook better. I can instead go the long way: cut the code, create a new cell, paste. But in Jupyter there's a shortcat for that I got used to. I'm creating this topic in case of many other people migrating from Jupyter to Marimo miss this feature.

<img width="276" alt="image" src="https://github.com/user-attachments/assets/f2b68f7f-5d44-444d-9ce2-5be133653856">

**Comment by mscolnick** [comment]
Hi, we already support Split Cell - you can find the shortcut in the "Keyboard Shortcuts" menu 
<img width="1125" alt="image" src="https://github.com/user-attachments/assets/a0e12ee0-f4e7-4bb7-9b27-69c3354b79cc">
